### PR TITLE
feat(symphony-templates): the hive templates

### PR DESCRIPTION
**Pull request by LeoArurault**
Playbook templates to create alerts on the hive from sekoia.io alerts

### CODE REVIEW

File: playbooks/templates/create_alert_on_the_hive_automatic.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "name": "Create Alert on TheHive automatically",
```
Comment (PierrickV) [inline playbooks/templates/create_alert_on_the_hive_automatic.json:2]:
```suggestion
    "name": "Automatically create an alert on TheHive",
```

File: playbooks/templates/create_alert_on_the_hive_automatic.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "description": "Create an alert on The Hive automatically when a new alert is created in the platform.",
```
Comment (PierrickV) [inline playbooks/templates/create_alert_on_the_hive_automatic.json:3]:
```suggestion
    "description": "Automatically create an alert on TheHive when a new alert is raised in SEKOIA.IO.",
```

File: playbooks/templates/create_alert_on_the_hive_manual.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{
+    "name": "Create manually Alert on TheHive",
```
Comment (PierrickV) [inline playbooks/templates/create_alert_on_the_hive_manual.json:2]:
```suggestion
    "name": "Manually create an Alert on TheHive",
```

File: playbooks/templates/create_alert_on_the_hive_automatic.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            "name": "Create the alert in The Hive",
```
Comment (PierrickV) [inline playbooks/templates/create_alert_on_the_hive_automatic.json:39]:
```suggestion
            "name": "Create the alert in TheHive",
```

File: playbooks/templates/create_alert_on_the_hive_manual.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    "description": "Create an alert on The Hive through the alert page.",
```
Comment (PierrickV) [inline playbooks/templates/create_alert_on_the_hive_manual.json:3]:
```suggestion
    "description": "Create an alert on TheHive through the alert page.",
```

File: playbooks/templates/create_alert_on_the_hive_manual.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+            "name": "Create the alert in The Hive",
```
Comment (PierrickV) [inline playbooks/templates/create_alert_on_the_hive_manual.json:39]:
```suggestion
            "name": "Create the alert in TheHive",
```

**Review by squioc — APPROVED** [review]
What is the difference between the two templates?

**Review by squioc — APPROVED** [review]

File: playbooks/templates/playbooks.json [hunk 7]
```diff
@@ -90,5 +90,19 @@
             "observable",
             "Fetch OSINT"
         ]
+    },
+    {
+        "file": "create_alert_on_the_hive_automatic.json",
+        "name": "Create Alert on TheHive automatically",
+        "workspace": "Operation Center",
+        "description": "Create an alert on The Hive automatically when a new alert is created in the platform.",
```
Comment (PierrickV) [inline playbooks/templates/playbooks.json:98]:
```suggestion
        "description": "Create an alert on TheHive automatically when a new alert is created in the platform.",
```

File: playbooks/templates/playbooks.json [hunk 8]
```diff
@@ -90,5 +90,19 @@
...
+        "tags": ["alert", "thehive"]
+    },
+    {
+        "file": "create_alert_on_the_hive_manual.json",
+        "name": "Create manually Alert on TheHive",
+        "workspace": "Operation Center",
+        "description": "Create an alert on The Hive through the alert page.",
```
Comment (PierrickV) [inline playbooks/templates/playbooks.json:105]:
```suggestion
        "description": "Create an alert on TheHive via the alert page.",
```

File: playbooks/templates/create_alert_on_the_hive_manual.json (hunk 6 above)
Comment (PierrickV) [inline playbooks/templates/create_alert_on_the_hive_manual.json:39]:
```suggestion
            "name": "Create the alert in TheHive",
```

**Review by PierrickV — APPROVED** [review]

**Pull request merged**